static/css/legacy.less

```diff
@@ -1176,7 +1176,45 @@ form {
     input[type=text],
     input[type=email],
     input[type=password] {
-      float: left;
+      &.floatform {
+        font-family: "Lucida Grande", "Trebuchet MS", Geneva, Helvetica, sans-serif;
```

Comment:
you'll need to use a variable for this.
Is it needed or is this still a problem with the rebase?